### PR TITLE
Compatibility with lastest Hyper version (3.x)

### PR DESCRIPTION
**Pull request by trystan2k**
Hi @zsol , how are you?

I read in the open issue that you are not maintaining this plugin anymore but I got some time today and was able to do some changes that make it compatible with Hyper.js 3.0

This PR has this changes, so if you could please have a look and let me know if you see any issue or have any doubt/comment, it would be great :)

Thank you !

### CODE REVIEW

File: src/PasteDialog.react.js [hunk 1]
```diff
@@ -83,16 +99,16 @@ export default class PasteDialog extends React.Component {
...
               width: '20px',
-              height: '20px',
+              height: '20px'
             }}>
               <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1000 1000" fill={config.foregroundColor}><g><path d="M720.9,396.3c-13.3-8.4-30.9-4.5-39.3,8.8L477.2,728.2L343.8,620.8c-12.2-9.9-30.2-7.9-40,4.5c-9.8,12.4-7.8,30.4,4.4,40.3l158.1,127.2c5.1,4.1,11.3,6.3,17.8,6.3c1.6,0,3.1-0.1,4.7-0.4c8-1.3,15-6,19.3-12.9l221.4-350C738,422.5,734.1,404.8,720.9,396.3z"/><path d="M768.8,141.4h-84.3v-26h-78.2C593.4,60.7,547,10,488.8,10c-58.2,0-108,50.7-120.8,105.4h-78.2v26h-58.5c-85.4,0-155,65.8-155,151.8v540.9c0,85.9,69.5,155.9,155,155.9h537.6c85.4,0,155-70,155-155.9V293.2C923.8,207.2,854.3,141.4,768.8,141.4z M419.9,166.9v-32.8c0-36.8,32.3-71.7,68.9-71.7c36.6,0,63.9,34.9,63.9,71.7v32.8h79.1v133.6H340V166.9H419.9z M869.4,834.1c0,54.4-46.5,102.8-100.5,102.8H231.2c-54.1,0-104.7-48.4-104.7-102.8V293.2c0-54.4,50.6-98.6,104.7-98.6h58.5v159.1h394.9V194.6h84.3c54.1,0,100.5,44.3,100.5,98.6L869.4,834.1L869.4,834.1z"/></g></svg>
             </button>
           </div>
-        </div>
-      </AriaModal>;
+        </div>;
```
Comment (zsol) [inline src/PasteDialog.react.js]: that semicolon is probably not necessary

**Comment by zsol** [comment]
Looks reasonable to me, but I have no way of trying it. :)

**Comment by trystan2k** [comment]
Hi @zsol !

I fixed the semicolon you pointed out and also during yesterday I found a bug that was duplicating the paste string if you have multiple tabs opened. 

If it is ok, how are the next step ? Once we merge the PR, how to publish a new version on NPM ? You did it manually ? If we release a new version in Github you will be able to publish it to NPM ?

Thank you !

**Comment by zsol** [comment]
Yep. Feel free to merge it, I'll release to npm

**Comment by trystan2k** [comment]
Ok. I will merge it.

**Pull request merged**